Parse/PFObject.h

```diff
@@ -813,6 +813,18 @@ NS_REQUIRES_PROPERTY_DEFINITIONS
  */
 - (BFTask *)deleteEventually;
 
+/*!
+ @abstract Reverts any changes to an object that were done after last successful save/fetch.
+ */
+ - (void)revert;
+
+/*!
+ @abstract Reverts any changes to an object's key that were done after last sucessful save/fetch.
```

Comment:
This should actually be

> Clears any changes to this object made since the last call to save.

Since it won't revert anything prior to the current operation set in the queue.